kubejs/server_scripts/enigmatica/kubejs/expert/recipetypes/occultism/crushing.js

```diff
@@ -0,0 +1,31 @@
+onEvent('recipes', (event) => {
+    if (global.isExpertMode == false) {
+        return;
+    }
+    const id_prefix = 'enigmatica:expert/occultism/crushing/';
+    const recipes = [
+        {
+            input: { item: 'upgrade_aquatic:embedded_ammonite' },
+            output: 'minecraft:nautilus_shell',
+            count: 3,
+            time: 100,
+            ignore_crushing_multiplier: false,
```

Comment:
ooh, multiplier on nautilus shell is a welcome addition :D

Reply:
Yeah. Seems like most people are defaulting to selling out ammonite, which is not really what I was expecting. And it's pretty rare compared to the nautilus mobs.